src/iframe-apprtc/index.html

```diff
@@ -14,11 +14,16 @@
   <meta http-equiv="X-UA-Compatible" content="chrome=1"/>
 </head>
 <body>
-  AppRTC in an &lt;iframe&gt; element: To be deprecated in M64 (~ Jan 2018) <br>
-  <iframe width="50%" height="600px" src="//appr.tc"></iframe>
-  <hr>
-  To test website still uses cross-origin iframes <br>
-  <iframe width="50%" height="600px" src="//appr.tc" allow="geolocation; microphone; camera"></iframe>
-  <hr>
+<table style="border:1px solid #000; witdh:100%;">
+  <tr>
+    <th>AppRTC in an &lt;iframe&gt; element: To be deprecated in M64 (~Jan 2018)</th>
+    <th>To test website still uses cross-origin IFRAMES</th>
+  </tr>
+  <tr>
+    <td style="border:1px solid #000; witdh:100%;"><iframe width="100%" height="600px" src="//appr.tc"></iframe></td>
```

Comment:
width is misspelled.

Reply:
Could you also try to use percentage for height, I remember it can be a bit fiddly with iframes but maybe things have improved?